docs/source/tutorial/calc_tutorial.rst

```diff
@@ -316,6 +354,45 @@ details. It is fairly easy to add your own scattering theory to HoloPy. See
 useful for other users, please consider submitting a `pull request
 <https://github.com/manoharan-lab/holopy/pulls>`_.
 
+Most of the scattering theories in `holopy` treat the fields on the detector as
+a (magnified) image of the fields at the focal plane. While these theories
+usually provide a good description of holograms of particles far above the
+focus, when the particle is near near the focus subtle optical effects can
+cause deviations between the recorded hologram and theories which do not
+specifically describe the effects of the lens. To deal with this, HoloPy
+currently offers two scattering theories which describe the effects of a perfect
+lens on the recorded hologram. Both of these scattering theories need information about the lens to make predictions, specifically the acceptance angle of the lens. The acceptance angle :math:`\beta` is related to the numerical aperture or NA of the lens by :math:`NA = n_f \sin \beta`, where :math:`n_f` is the refractive of the immersion fluid. For more details on this, see our papers `here<https://www.osapublishing.org/oe/abstract.cfm?uri=oe-28-2-1061>`_ and `here<url>`_.
+
+The :class:`.Lens` theory allows `holopy` to include the effects of a perfect
+objective lens with any scattering theory. The Lens theory works by wrapping a
+normal scattering theory. For instance, to calculate the image of a sphere in
+an objective lens with an acceptance angle of 1.0, do
```

Comment:
can we explicitly relate acceptance angle to NA, since that is what is printed on the objective?

Reply:
should be clarified by d3bbdce